bin/ingest_single_footprint.sh

```diff
@@ -48,7 +47,6 @@ cat << EOF
     min_zoom=greatest(0, approximate_zoom - 4),
     max_zoom=20,
```

Comment:
I meant "shouldn't" … won't that result in pixels that are too large? Or is gmted/etopo the only one with that problem?